andela/ah-frontend-realers
# #166187807 Redux Setup

#### What does this PR do?
* Setting redux and appropriate file structure
#### Description of task completed
* Set redux and an appropriate file structure
#### How should this be manually tested on your local machine
* Do a `git fetch origin ch-redux-setup-166187807`
* Do a `git checkout ch-redux-setup-166187807`
* Install all the dependencies using `npm install`
* install [redux dev tools](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd?hl=en) in chrome
* Run the app using `npm start`
* Right-click on your browser and inspect.
* Click on `redux >> raw`
* You should see an empty state object




[Maintains #166187807](https://www.pivotaltracker.com/n/projects/2350340/stories/166187807)
